usecases/MinimumWorkingExample/perform_calibration_prediction_post_dodo.py

```diff
@@ -0,0 +1,85 @@
+# Atul : this is a sandbox file for the calibration, please let it be here
```

Comment:
I personally would really want to try to keep our repository, at least the main branch, as clean as possible.
You can keep the file on your system, but I would not want to commit it in the main branch.
This seems like a copy and past of the functions you have defined in the doit.
I would suggest, if you really need to use these functions outside of doit, to move them to the lebedigital/calibration directory and then include them in the doit directly. If you want to change something and test is separately, you would now have to change two files.

Reply:
ya alright. make sense